<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCutSample</class>
 <widget class="QDialog" name="DialogCutSample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>42</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入影像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>82</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>存放路径</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxOutput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInput">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutput">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ImageSize">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>图块大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Overlay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>重叠度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SizeX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X方向</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_OverlayX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X方向</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SizeX">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_OverlayX">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_SizeY">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y方向</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SizeY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_OverlayY">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y方向</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_overlayY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_calculating">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>202</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>正在裁剪</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
